src/components/page-step/index.tsx

```diff
@@ -0,0 +1,279 @@
+        />
+        {!this.props.step ? (
+          <Typography>No step…</Typography>
+        ) : (
+          <>
+            <section className="grid grid--gap-medium">
+              <Typography variant="h4">Overview</Typography>
+              <div className="grid grid--gap-medium grid--overview-columns">
+                <div className="grid grid--gap-medium">
+                  <Typography>
+                    Pipeline Step <strong>{step.status.toLowerCase()}</strong>{' '}
+                  </Typography>
+                  <Typography>
+                    {getExecutionState(step.status)} Step{' '}
+                    <strong>{getPipelineStepTitle(step.name)}</strong>
+                  </Typography>
+                </div>
+                {step.started && (
+                  <div className="grid grid--gap-medium">
+                    <Typography>
+                      Started{' '}
+                      <strong>
+                        <RelativeToNow time={step.started} />
+                      </strong>
+                    </Typography>
+                    {step.ended ? (
+                      <Typography>
+                        Step took{' '}
+                        <strong>
+                          <Duration start={step.started} end={step.ended} />
+                        </strong>
+                      </Typography>
+                    ) : (
+                      <Typography>
+                        Duration so far is{' '}
+                        <strong>
+                          <Duration start={step.started} end={this.state.now} />
+                        </strong>
+                      </Typography>
+                    )}
+                  </div>
+                )}
+              </div>
+            </section>
+            {this.props.step.scan &&
+              this.props.step.scan.status === ScanStatus.Success && (
+                <section className="grid grid--gap-medium">
+                  <Typography variant="h4">Vulnerabilities</Typography>
+                  <ScanOutput
+                    appName={this.props.appName}
+                    jobName={this.props.jobName}
+                    stepName={this.props.step.name}
+                  />
+                </section>
+              )}
+            {this.props.stepName === 'run-pipelines' &&
+              (this.props.pipelineRuns && this.props.pipelineRuns.length > 0 ? (
+                <section className="step-log">
+                  <Typography
+                    variant="h4"
+                    className={`pipeline-run-header-absolute'`}
+                  >
+                    Environment pipelines
+                  </Typography>
+                  <AsyncResource
+                    resource="PIPELINE_RUNS"
+                    resourceParams={[appName, jobName]}
+                  >
+                    <PipelineRuns
+                      appName={this.props.appName}
+                      jobName={this.props.jobName}
```

Comment:
use variables declared above instead
```suggestion
                      appName={appName}
                      jobName={jobName}
```